src/components/consul/selectors/DocumentSelector.js

```diff
@@ -60,6 +60,66 @@ export const DocumentSelector = ({ control, field, item }) => {
     setInfoAndErrorText(deleteArrayItem(infoAndErrorText, index));
   };
 
+  if (isVolunteer) {
+    return (
+      <>
+        <Input {...item} control={control} hidden name={name} value={JSON.stringify(value)} />
+        <RegularText smallest placeholder>
+          {infoText}
+        </RegularText>
+
+        <Button
+          title={buttonTitle}
+          invert
+          onPress={async () => {
+            const { name: title, size, uri, mimeType } = await selectDocument();
+
+            if (!uri) return;
+
+            /* the server does not support files more than 10MB in size. */
+            const errorText = size > 10485760 && texts.volunteer.imageGreater10MBError;
+
+            setDocumentsAttributes([...documentsAttributes, { uri, mimeType }]);
+
+            setInfoAndErrorText([
+              ...infoAndErrorText,
+              {
+                errorText,
+                infoText: `${title}`
+              }
+            ]);
+          }}
+        />
+
+        {value
+          ? JSON.parse(value).map((item, index) => (
+              <View key={index}>
+                {!!infoAndErrorText[index]?.errorText && (
+                  <RegularText smallest error>
+                    {infoAndErrorText[index].errorText}
+                  </RegularText>
+                )}
+
+                <View style={styles.volunteerImageView}>
+                  {!!infoAndErrorText[index]?.infoText && (
+                    <RegularText style={{ width: '90%' }} numberOfLines={1} small>
```

Comment:
can you move this and same in `ImageSelector` to styles please

![image](https://user-images.githubusercontent.com/1942953/183452912-c0c94aa2-893d-412a-9eef-08de9726c441.png)

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/606/commits/ba048ed71a501a766f49e70c8bc6dbc33e9c673f
https://github.com/ikuseiGmbH/smart-village-app-app/pull/606/commits/8f4269d2017e14fcc104932249cffd5f1c467b34